docs/developing_iree/profiling_cpu_events.md

```diff
@@ -0,0 +1,13 @@
+# CPU cache and other CPU event profiling
+
+## Android
+
+On Android we can use `simpleperf`. It's preinstalled on current Android `userdebug` images, and part of the Android NDK.
+
+TODO write me
```

Comment:
Link to https://developer.android.com/ndk/guides/simpleperf for now?

Reply:
done!